packages/store-ui/src/atoms/Radio/Radio.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef } from 'react'
+import type { InputHTMLAttributes } from 'react'
+
+export interface RadioProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, 'type'> {
+  testId?: string
+}
+
+const Radio = forwardRef<HTMLInputElement, RadioProps>(function Radio(
```

Comment:
What do you think about adding the `forwardRef` in the `default export` below? Then you can just declare the function normally, as I did here: https://github.com/vtex/faststore/pull/894/files#diff-7120a0a181731ef8d2182a6e50663e183f52b42824d9b9e9e1fc0fe009f35264R9.

I find that including the `forwardRef` right here in the definition makes it a bit polluted with type anotations and also defining the `Radio` function inside another function's call.

Reply:
@igorbrasileiro, @Gmantiqueira and @icazevedo I would also like your input here since we should probably choose a pattern to follow regarding this.

Reply:
Yeah, it's a little bit polluted but we need to pay attention to types to not write wrong. But overall I don't have an opinion about it.

Reply:
I think it's not is not a problem, because if we add the forwardRef on export default, we need to add `ForwardedRef<HTMLDivElement>` to ref attribute on the component.

Reply:
What do you think about creating a Discussion about that? because Radio is not the only component like this.

Reply:
Yeah, we still need to add the type as a function argument, but it's easier to read the function definition since now you're only defining a function that takes two arguments. It seems way cleaner to me.

Reply:
Yes, I understand. Just I think we need to make this change on all components (in another pr) but before that, I think we need to take the opinion of the other developers in a discussion.